cfgov/cal/views.py

```diff
@@ -31,6 +33,31 @@
        PDFreactor = None
 
 
+class FilterErrorList(forms.utils.ErrorList):
+    def __str__(self):
+        return '\n'.join(str(e) for e in self)
+
+class FilterCheckboxList(forms.MultipleChoiceField):
+    def validate(self, value):
+        if self.required and not value:
+            msg = self.error_messages['required']
+            if self.label and '%s' in msg:
+                msg = msg % self.label
+            raise forms.ValidationError(msg, code='required')
+        return value
+
+class FilterDateField(forms.DateField):
+    def clean(self, value):
+        if value:
+            try:
+                value = get_date_string(value)
+            except ValueError:
+                msg = self.error_messages['invalid']
+                if '%s'in msg:
+                    msg = msg % value
+                raise forms.ValidationError( msg , code='invalid')
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2263/commits/8b1241190c12d6ca8aef3094ad43dcefe3459d7e.